packages/sync/src/provider.js

```diff
@@ -5,16 +5,23 @@
 /**
  * External dependencies
  */
+// @ts-ignore
+import { removeAwarenessStates as removeAwarenessStatesFromProtocol } from 'y-protocols/awareness';
 import * as Y from 'yjs';
 
 /** @typedef {import('./types').ObjectType} ObjectType */
 /** @typedef {import('./types').ObjectID} ObjectID */
 /** @typedef {import('./types').ObjectConfig} ObjectConfig */
 /** @typedef {import('./types').ConnectDoc} ConnectDoc */
 /** @typedef {import('./types').SyncProvider} SyncProvider */
+/** @typedef {import('./types').AwarenessEventListener} AwarenessEventListener */
+/** @typedef {import('./types').PendingAwarenessSetup} PendingAwarenessSetup */
+/** @typedef {import('y-protocols/awareness').Awareness} Awareness */
 /** @typedef {import('./types').UndoManager} UndoManager */
 /** @typedef {import('./types').HistoryRecord} HistoryRecord */
 
+const AWARENESS_DOC_TYPE = 'postType/Posts';
```

Comment:
Not a blocker rn, but for a future PR: Should this be just `postType` similar to the undo manager so that it works for pages and other CPTs as well rather than just posts?

Reply:
Probably, good call-out.